apps/payments/next/styles/global.css

```diff
@@ -1 +1,5 @@
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
```

Comment:
This was necessary to get rid of some errors related to `postcss-import`.

https://tailwindcss.com/docs/using-with-preprocessors#build-time-imports